webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -77,7 +77,7 @@
                                 <th scope="row">Signature</th>
                                 <td>
                                     {{ report.signature }}
-                                    <a href="{{ url('crashstats:report_list') }}?product={{ report.product }}&amp;signature={{ report.signature|urlencode }}" class="sig-overview" title="View more reports of this type">More Reports</a>
+                                    <a href="{{ url('signature:signature_report') }}?product={{ report.product }}&amp;signature={{ report.signature|urlencode }}&amp;version={{ report.version|urlencode }}" class="sig-overview" title="View more reports of this type">More Reports</a>
```

Comment:
Why do you add the version if it wasn't there before? I don't think we want to change the way this works, just link to the new Signature report.